xml/src/main/scala/akka/stream/alpakka/xml/Xml.scala

```diff
@@ -178,6 +179,63 @@ object Xml {
   /**
    * Internal API
    */
+  private[xml] class StreamingXmlWriter(strEncoding: String) extends GraphStage[FlowShape[ParseEvent, ByteString]] {
```

Comment:
Have you considered using `Charset` instead of String. Could be a better choice even though `XMLOutputFactory` accepts charset names only.

Reply:
I think I had changed my mind because of the charset name for XMLOutputFactory but you are right it would be better.